packages/lesswrong/server/spotlightCron.ts

```diff
@@ -45,7 +47,7 @@ addCronJob({
     // But doing it this way prevents a slow "walk" due to the cron job interval...
     // ...ensuring the promotion happens at around the same time, and each period length is still roughly constant
     const newPromotionDate = new Date(lastPromotionDate);
-    newPromotionDate.setDate(lastPromotionDate.getDate() + ACTIVE_DAYS);
+    newPromotionDate.setDate(lastPromotionDate.getDate() + DEFAULT_ACTIVE_DAYS);
```

Comment:
I think we need to use the same value as [here](https://github.com/ForumMagnum/ForumMagnum/pull/5766/files#diff-6197d7cb81e27c8021e3037f7c83cc9fa7e1b299f30ff99d775978cf590a9929R29), or we'll set the next spotlight's `lastPromotedAt` date to a date in the future if the current one's duration is shorter than the default.